staging/kubefed/values.yaml

```diff
@@ -33,10 +33,19 @@ controllermanager:
     adoptResources:
   ## Value of feature gates item should be either `Enabled` or `Disabled`
   featureGates:
-    PushReconciler:
-    SchedulerPreferences:
-    CrossClusterServiceDiscovery:
-    FederatedIngress:
+    featureGates:
+      CrossClusterServiceDiscovery: "Disabled"
+      FederatedIngress: "Enabled"
+      PushReconciler: "Enabled"
+      SchedulerPreferences: "Disabled"
+
+  controllermanager:
+    annotations: {}
+
+  webhook:
```

Comment:
`controllermanager.webhook` is also a bit weird - would be better to have webhook toplevel.

Reply:
> `controllermanager.webhook` is also a bit weird - would be better to have webhook toplevel.

As i answered above, this is just unfortunate, not weird. If i were to change this chart i would not have a subchart in this way, among other changes.

Reply:
The use of subcharts like this is to be able to enable/disable controllermanager separately - I don't see that as an abuse, but assume they are doing so for development purposes for some reason.

Webhook isn't part of controllermanager - it's a separate deployment/component in this environment so it is weird to nest it under controllermanager :)

Reply:
it is under the controllermanager subchart, as such it belongs nested and not top level.

Reply:
I know, what I'm saying is that is odd - controllemanager and webhook are two different components and if a subchart is used for controllermanager then it makes sense to make a separate subchart for webhook.

Reply:
> I know, what I'm saying is that is odd - controllemanager and webhook are two different components and if a subchart is used for controllermanager then it makes sense to make a separate subchart for webhook.

with the internal logic of moving the controller to a separate subchart, yes i agree with you, it is odd. 

I have little idea why this was done this way unless the intent was that the subchart can stand alone? 

Regardless I understand your point and the reason for doing the PR as you have added. My only concern was the timing and what upstream chart work would be accepted... I guess i need a point of contact and get some bearing there.